modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -326,3 +326,50 @@ function oe_theme_helper_post_update_20017() {
   $image_style->addImageEffect($effect);
   $image_style->save();
 }
+
+/**
+ * Add banner image styles.
+ */
+function oe_theme_helper_post_update_40001(): void {
+  $image_styles = [
+    'oe_theme_large_banner' => [
+      'label' => 'Large banner',
+      'width' => '996',
+      'height' => '256',
+    ],
+    'oe_theme_medium_banner' => [
+      'label' => 'Medium banner',
+      'width' => '768',
+      'height' => '332',
+    ],
+    'oe_theme_small_banner' => [
+      'label' => 'Small banner',
+      'width' => '480',
+      'height' => '160',
+    ],
+  ];
+  foreach ($image_styles as $style_id => $style_data) {
+    $style = \Drupal::entityTypeManager()->getStorage('image_style')->load($style_id);
```

Comment:
Let's instantiate a `$storage = \Drupal::entityTypeManager()->getStorage('image_style');` outside of the foreach.